Update Movement

Cdttn8 opened the pull request:
Added new camera and fixed animation as well as left and right movement

cdttn99 committed:
Update Movement

Added new camera and fixed animation as well as left and right movement

cdttn99 committed:
Update movement

Added a place to move around. Dodge animation. Removed temp scene

NamesAreHard6600 reviewed (changes requested):
Please fix the attached issue. 
Should be good after that. 
Thanks!

cdttn99 committed:
Update Movement

Fixed arrows bouncing now properly hits things its supposed to and gets destroyed. Also moved scenes into parent directory to avoid confusion

NamesAreHard6600 reviewed (approved):
Lgtm!
